iocore/eventsystem/I_EventProcessor.h

```diff
@@ -30,15 +30,15 @@
 #include "I_Event.h"
 
 #ifdef TS_MAX_THREADS_IN_EACH_THREAD_TYPE
-const int MAX_THREADS_IN_EACH_TYPE = TS_MAX_THREADS_IN_EACH_THREAD_TYPE;
+constexpr int MAX_THREADS_IN_EACH_TYPE = TS_MAX_THREADS_IN_EACH_THREAD_TYPE;
 #else
-const int MAX_THREADS_IN_EACH_TYPE = 3072;
+constexpr int MAX_THREADS_IN_EACH_TYPE = 3072;
 #endif
 
 #ifdef TS_MAX_NUMBER_EVENT_THREADS
-const int MAX_EVENT_THREADS = TS_MAX_NUMBER_EVENT_THREADS;
+constexpr int MAX_EVENT_THREADS = TS_MAX_NUMBER_EVENT_THREADS;
 #else
-const int MAX_EVENT_THREADS = 4096;
+constxpr int MAX_EVENT_THREADS = 4096;
```

Comment:
I'll drop that in when I get a fix for the FreeBSD crash. But first I need FreeBSD somewhere.